### ISSUE TITLE
Is route deprecated?

### ISSUE THREAD

**Issue by davidmorgan** [justinfagnani/route#92]
I heard rumour route is deprecated in favour of shelf -- is this the case?

If so, a mention in the README would be good :)

Thanks!

**Comment by enyo** [comment]
Yes... and now it's deprecated on pub, but still no mention in the README and what should be used instead. That would be great.
